src/synthesize.jl

```diff
@@ -92,59 +92,49 @@ function synthesize(atm::ModelAtmosphere, linelist, λs::AbstractRange; metallic
     abundances = get_absolute_abundances(metallicity, abundances)
     MEQs = molecular_equilibrium_equations(abundances, ionization_energies, partition_funcs, 
                                            equilibrium_constants)
-    n_dicts = map(atm.layers) do layer
-         molecular_equilibrium(MEQs, layer.temp, layer.number_density, layer.electron_number_density)
-    end
-    number_densities = Dict([spec=>[n[spec] for n in n_dicts] for spec in keys(n_dicts[1])])
 
     #the absorption coefficient, α, for each wavelength and atmospheric layer
     α_type = typeof(promote(atm.layers[1].temp, length(linelist) > 0 ? linelist[1].wl : 1.0, λs[1], 
                             metallicity, vmic, abundances[1])[1])
     α = Matrix{α_type}(undef, length(atm.layers), length(λs))
 
-    #Calculate the continuum absorption over cntmλs, which is a sparser grid, then construct an
-    #interpolator that can be used to approximate it over a fine grid.
-    sorted_cntmνs = c_cgs ./ reverse(cntmλs)
-    α_cntm = map(zip(atm.layers, n_dicts)) do (layer, ns)
-        α_cntm_vals = total_continuum_absorption(sorted_cntmνs, layer.temp,
-                                                 layer.electron_number_density,
-                                                 ns, partition_funcs)
-        reverse!(α_cntm_vals)
-        LinearInterpolation(cntmλs, α_cntm_vals) # LinearInterpolation needs ranges to be increasing
-    end
+    sorted_cntmνs = c_cgs ./ reverse(cntmλs) #frequencies at which to calculate the continuum
 
+    α_cntm = Vector(undef, length(atm.layers))     #vector of continuum-absorption interpolators
+    α5 = Vector{α_type}(undef, length(atm.layers)) #reference absorption at each layer
```

Comment:
```suggestion
    α5 = Vector{α_type}(undef, length(atm.layers)) #each layer's absorption at reference λ (5000 Å)
```
I know this seems pedantic, but I think this tweak should avoid confusion (I also confirmed that the line is still under 100 characters)